nginx/default.conf.template

```diff
@@ -10,12 +10,11 @@ server {
     server_name localhost;
     # add hosts your domain for local requests
     # server certificate
-    ssl_certificate     /etc/nginx/certs/server-cert.pem;
+    ssl_certificate     /etc/nginx/certs/server-cert-bundle.pem;
     ssl_certificate_key /etc/nginx/certs/server-key.pem;
     # CA certificate
-    ssl_client_certificate /etc/nginx/certs/ca-cert.pem;
-    # need to validate client certificate(if this flag optional_no_ca it won't validate client certificates)
-    ssl_verify_client optional_no_ca;
+    ssl_client_certificate /etc/nginx/certs/client-ca-cert.pem;
+    ssl_verify_client optional;
```

Comment:
@bensummers client certificates will be verified if present. It's not set to `on` as we have some endpoints that don't require mtls